boards/xtensa/esp32/esp32-devkitc/src/esp32-devkitc.h

```diff
@@ -245,7 +245,7 @@ int board_ws2812_initialize(
  *
  ****************************************************************************/
 #ifdef CONFIG_ESP32_OPENETH
-int esp32_openeth_initialize(void);
+int xtensa_openeth_initialize(void);
```

Comment:
should not name to xtensa_ since it's espxxx specific.

Reply:
what do you suggest we name it?

both esp32 and esp32s3 are wrong names because the driver is the same for both chips.

Reply:
> what do you suggest we name it?
> 

esp_common? @tmedicci @fdcavalcanti please take a look.

> both esp32 and esp32s3 are wrong names because the driver is the same for both chips.

yes, but put into common is also wrong since this driver can just work with esp chipset.

Reply:
I guess we can put in common/espressif

I will update the PR

Reply:
Hello. @xiaoxiang781216 Tiago is out for a couple of weeks.
@casaroli pleasee please put under common/espressif. That's where all common Xtensa Espressif source goes.

Reply:
done